$OpenBSD: patch-src_occtl-time_c,v 1.1 2015/05/25 20:57:11 sthen Exp $

time_t is 64 bits on all OpenBSD (and NetBSD) arch; cast time values
to a specific-width type to avoid problems on 32-bit arch

--- src/occtl-time.c.orig	Mon May 25 21:50:10 2015
+++ src/occtl-time.c	Mon May 25 21:51:00 2015
@@ -43,17 +43,17 @@ void print_time_ival7(time_t t1, time_t t2, FILE * fou
 	
 	if (t >= 48 * 60 * 60)
 		/* 2 days or more */
-		fprintf(fout, _("%2ludays"), t / (24 * 60 * 60));
+		fprintf(fout, _("%2lludays"), (long long)t / (24 * 60 * 60));
 	else if (t >= 60 * 60)
 		/* 1 hour or more */
 	        /* Translation Hint: Hours:Minutes */
-		fprintf(fout, _("%2luh:%02um"), t / (60 * 60),
+		fprintf(fout, _("%2lluh:%02um"), (long long)t / (60 * 60),
 			(unsigned)((t / 60) % 60));
 	else if (t > 60)
 		/* 1 minute or more */
 	        /* Translation Hint: Minutes:Seconds */
-		fprintf(fout, "%2lum:%02us", t / 60, (unsigned)t % 60);
+		fprintf(fout, "%2llum:%02us", (long long)t / 60, (unsigned)t % 60);
 	else
 	        /* Translation Hint: Seconds:Centiseconds */
-		fprintf(fout, _("%5lus"), t);
+		fprintf(fout, _("%5llus"), (long long)t);
 }
